src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -172,6 +169,11 @@
     <Compile Include="System\Transactions\DistributedTransactionPermission.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermissionAttribute.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
```

Comment:
Do we really need to be building netstandard version of this?

Reply:
@ericstj ?

Reply:
Yes we need it for other frameworks like UAP. All of our packages need to have a matching ref and src implementation for NS in order to be restorable (used).

Reply:
This change should be done for UAP as well (it is partially done in this PR already). We structure UAP and NETCOREAPP same way. UAP should not be using netstandard version of the package after this change.